angularSTEP/src/app/email/email.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="example-container">
+  <div class="form-container">
+    <h2>Sign In</h2>
+    <span class="error" *ngIf="error">{{ error }}</span>
+    <form [formGroup]="logInForm" (submit)="onSubmit()">
```

Comment:
In Angular [squareBrackets] and (parenthesis) are two different things, yes. In the code snippet you highlighted, the way forms work in Angular is by putting the name of the formGroup within [squareBrackets]. The [squareBrackets] are an input field. If you were to have a component, you could pass in an input for a variable. The (parenthesis) are statements and declare an event that is binded to the element. So when the form is "(submit)"ed it will call the function.